src/overlays/actors/ovl_En_Rl/z_en_rl.h

```diff
@@ -6,9 +6,20 @@
 
 struct EnRl;
 
+typedef void (*EnRlActionFunc)(struct EnRl*, GlobalContext*);
+typedef void (*EnRlDrawFunc)(struct EnRl*, GlobalContext*);
+
```

Comment:
I'm not sure but since these aren't used in the header they may be better located in the .c ?

Reply:
as of now we do keep these in the header. keeping it in C definitely makes sense though.
@Roman971 thoughts on cases like these when the typedefs arent needed externally/in headers? would there ever be a reason where you would want these typedefs in the header for modding purposes?

Reply:
I'd say they should be in the header for consistency and modding purposes